packages/ariakit/src/checkbox/__examples__/checkbox-group/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Checkbox, useCheckboxState } from "ariakit/checkbox";
+import { Group, GroupLabel } from "ariakit/group";
+
+import "./style.css";
+
+export default function Example() {
+  const checkbox = useCheckboxState({ defaultValue: [] });
+
+  return (
+    <Group className="group">
+      <GroupLabel className="group-label">Your favorite fruits</GroupLabel>
+      {checkbox.value.length > 0 ? <p>{checkbox.value.join(", ")}</p> : <br />}
```

Comment:
I see value in demonstrating how to access the checkbox state value in the example, but I'm not sure if this is a good example (that is, a real-world use case). I guess we can leave this out until we think of a better way.

Also removing the group class names.
```suggestion
    <Group>
      <GroupLabel>Your favorite fruits</GroupLabel>
```